mozartdata/transforms/dim/old_to_new_sku_map.sql

```diff
@@ -0,0 +1,37 @@
+/*
+ Purpose: Map old skus to new skus and new upcs
+ Granularity: One row per old sku
+ Primary key column: old_sku
+ Joining:
+    To old products: use column old_sku to sku
+    To new products: use column new_sku to sku
+ Notes: This data is human input. Please consider this when troubleshooting.
+ */
+
+select
+    upc.display_name -- Not recommended for joining but feasible for some tables
+  , upc.old_sku -- PRIMARY KEY, most recommended for joining
```

Comment:
I like the most important column first. 
```suggestion
    upc.old_sku -- PRIMARY KEY, most recommended for joining
  , upc.display_name -- Not recommended for joining but feasible for some tables
```